# HG changeset patch
# User Ben Abbott  <bpabbott@mac.com>
# Date 1332635544 14400
#      Sat Mar 24 20:32:24 2012 -0400
# Node ID 351ca094580b47dcb513993643c15068a94823d8
# Parent  7ce925166af650c6470653f17ae5f7ee13c6ad19
Add terminating semicolon.

* __go_draw_axes__.m (no_super_sub_scripts): Add terminating semicolon
missing since changeset 0b94080d2b0f.

diff --git a/scripts/plot/private/__go_draw_axes__.m b/scripts/plot/private/__go_draw_axes__.m
--- a/scripts/plot/private/__go_draw_axes__.m
+++ b/scripts/plot/private/__go_draw_axes__.m
@@ -2319,17 +2319,17 @@ function str = no_super_sub_scripts (str
     for m = 1 : numel(labels)
       n1 = strfind (labels{m}, sprintf ("\\%s", marker));
       n2 = strfind (labels{m}, marker);
       if (! isempty (n1))
         n1 = n1 + 1;
         n2 = setdiff (n2, n1);
       end
       for n = numel(n2):-1:1
-        labels{m} = [labels{m}(1:n2(n)-1), "\\", labels{m}(n2(n):end)]
+        labels{m} = [labels{m}(1:n2(n)-1), "\\", labels{m}(n2(n):end)];
       endfor
     endfor
   endfor
   if (iscellstr (str))
     str = labels;
   else
     str = char (labels);
   endif
